torchtext/csrc/vocab.cpp

```diff
@@ -152,8 +152,19 @@ _concat_tokens(std::vector<std::shared_ptr<StringList>> chunk_tokens,
     }
   }
 
-  // create tokens list and stoindex map
   int64_t index = 0;
+  // insert unk_token if not present
```

Comment:
Are those lines duplicate? I thought we add the `unk_token` in L76 of `vocab.py`.

Reply:
These are not duplicate. The C++ one is for loading a vocab from a file. The python one is creating a Vocab with a OrderedDict as the input.

Reply:
Right. I forgot another path to load vocab.